sdk/openai/azure-ai-openai/src/test/java/com/azure/ai/openai/OpenAIAsyncClientTest.java

```diff
@@ -242,6 +242,21 @@ public void testGetEmbeddings(HttpClient httpClient, OpenAIServiceVersion servic
         });
     }
 
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("com.azure.ai.openai.TestUtils#getTestParameters")
+    public void getEmbeddingsWithSmallerDimensions(HttpClient httpClient, OpenAIServiceVersion serviceVersion) {
```

Comment:
We should have test cases for `getEmbeddingAsString()` method

Reply:
Tests in https://github.com/Azure/azure-sdk-for-java/pull/39296/files#diff-98ebf74b657b3f63b0c36d212b741da5adec3261dcc6fa6666a4d482dd6595a6R554-R557